### PR TITLE
Add post on version 0.3.1 release

### PR DESCRIPTION
**Pull request by namurphy**
Here is a draft blog post to announce our 0.3.1 release.  Suggestions are welcome!   I also updated a link in an earlier blog post.

### CODE REVIEW

**Comment by rocco8773** [comment]
will close issue #46

**Comment by StanczakDominik** [comment]
I think we kind of forgot to merge this one... >_< I'll do it now, I guess!

**Comment by namurphy** [comment]
Huh, it's really unlike me to create a pull request and then forget about it for several months.  :joy_cat: Thanks for merging!

File: web/posts/plasmapy-v031-release.rst [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+<http://docs.plasmapy.org/en/stable/about/release_notes.html#version-0-3-1>`_.
+This release includes two new subpackages: ``plasmapy.formulary``,
+which consolidates much of the functionality that was formerly in
+``plasmapy.physics``, ``plasmapy.mathematics``, and
+``plasmapy.transport``; and ``plasmapy.simulation``, which contains
+our particle tracker and will be the place to consolidate tools
+related to plasma simulations.  These changes are part of an ongoing
+plan to reorganize PlasmaPy's subpackage structure that will continue
+in version 0.4.  In order to install PlasmaPy, please refer to the
+`installation instructions <>`_ on `PlasmaPy's documentation page
```
Comment (rocco8773) [inline web/posts/plasmapy-v031-release.rst]: missing link

**Comment by rocco8773** [comment]
I just noticed the post is not building correctly. (https://www.plasmapy.org/news/)

[It looks like the "installation instructions" is missing its link.](https://github.com/PlasmaPy/plasmapy.github.io/pull/43/files#r401052921)

**Pull request merged**